flocx_market/api/bid.py

```diff
@@ -1,25 +1,31 @@
 from flask_restful import Resource
 from flask import request, g
```

Comment:
Really, flask? You're going to name a public API attribute `g`?

Reply:
I had a very similar reaction :)